_posts/2016-05-07-sensitivity-analysis.md

```diff
@@ -1,6 +1,85 @@
 ---
 layout: post
 title: Sensitivity Analysis
-img: image-5.png
+img: sensitivity.png
 ---
 
+Sensitivity analysis helps to the study of how the variations in the outputs of a numerical or 
+physical system can be assigned to different sources of variation in its inputs.
+
+Understanding the variance of the outputs of a system in terms of the contributions of the inputs is a key aspect of
+uncertainty quantification: it helps to test the robustness of a system, design strategies for uncertainty reduction
+and helps to calibrate physical models to the real world by improving their interpretability.
+
+Sensitivity analysis methods are divided in two main classes: *local* methods, where changes in the output are studied for
+specific values of the inputs, and *global* methods where the total variability of the outputs is assigned to each input variable. Given a physical model of a system, for instance a climate model simulator, local sensitivity analysis is carried out using gradients.
+However, if the physical model is expensive-to-evaluate approximating gradients becomes intractable. Using an emulator of the physical model, like
+a Gaussian process, and using its gradients is an effective alternative.
+
+A similar situation arises when doing global sensitivity analysis. The core element in global sensitivity analysis are the Sobol indices, also called *main effects* [[1,2](#references-on-sensitivity-analysis)].
+They are a measure of “first order sensitivity” of each input variable. They account for the proportion of variance of the output explained by
+changing each variable alone while marginalizing over the rest. The *total effects* are also common [[3](#references-on-sensitivity-analysis)]. They account for the contribution to the
+output variance of each variable but including all variance caused by the variable alone and all its interactions of any order of the inputs. Both indices can be easily computed in Emukit.
+
+To illustrate how to do global sensitivity analysis of a simulator we use the [Ishigami function](https://www.sfu.ca/~ssurjano/ishigami.html).
+This function has three continuous inputs and depends on two parameters that we fix in this example.
+
+```python
+from emukit.test_functions.sensitivity import Ishigami
+
+ishigami = Ishigami(a=5, b=0.1)
+target_simulator = ishigami.fidelity1
+```
+
+Emukit allows to compute both, the main and total effects using Monte Carlo with the Saltelli estimators [[4](#references-on-sensitivity-analysis)].
+Instead of computing the indices by calling the simulator, we wi learn an emulator first and we compute the indices using *cheap* calls to this model. We first define the parameter space:
```

Comment:
```suggestion
Instead of computing the indices by calling the simulator, we learn an emulator first and we compute the indices using *cheap* calls to this model. We first define the parameter space:
```